source/postprocess/basic_statistics.cc

```diff
@@ -44,6 +44,9 @@ namespace aspect
               const MaterialModel::Simple<dim> &material_model
                 = dynamic_cast<const MaterialModel::Simple<dim> &>(this->get_material_model());
 
+              MaterialModel::MaterialModelInputs<dim> in (1, this->n_compositional_fields());
+              MaterialModel::MaterialModelOutputs out (1, this->n_compositional_fields());
```

Comment:
Oh, I see now -- you can't call the old functions any more since they're gone? Right?